docker/build.sh

```diff
@@ -34,23 +34,119 @@ trap cleanup EXIT
 REMOVE_CONTAINER="--rm"
 [[ -n ${KEEP_CONTAINER} ]] && unset REMOVE_CONTAINER
 
-case "$1" in
-  -h | --help ) ${DIR}/scripts/build.sh -h; exit 0 ;;
-esac
+function usage {
+  echo -e "\nUsage: `basename $0` [OPTIONS]" \
+          "\nBuild the current local TinkerPop project in a Docker container." \
+          "\n\nOptions are:\n" \
+          "\n\t-t,  --tests              run standard test suite for modules except non-Java GLVs" \
+          "\n\t-i,  --integration-tests  run integration tests for modules except non-Java GLVs" \
+          "\n\t-n,  --neo4j              include Neo4j" \
+          "\n\t-a,  --all                run tests for all non-Java GLVs and gremlin-console" \
+          "\n\t-go, --golang             run tests for Go GLV" \
+          "\n\t-py, --python             run tests for Python GLV" \
+          "\n\t-dn, --dotnet             run tests for Dotnet GLV" \
+          "\n\t-js, --javascript         run tests for JavaScript GLV" \
+          "\n\t-c,  --console            run tests for gremlin-console" \
+          "\n\t-m,  --minimal            build only the modules required to test GLVs (driver, server, test, console, neo4j)" \
```

Comment:
This is slightly confusing to understand. Perhaps give better description, or remove it.